packages/manager/modules/pci/src/components/project/empty/empty.component.js

```diff
@@ -0,0 +1,13 @@
+import controller from './empty.controller';
```

Comment:
Wouldn't `controller.js` suffice as a file name ? I'd like to know as I use modules too :)

Reply:
Instead of have too many file named exactly the same across the entire repository, then the closest module/folder name is used as a prefix I think.

Reply:
Indeed, they are prefixed that way in order to be easier to differenciate. Imo, it gets quite hard and messy when you start having all your files named `controller`

Reply:
It's yours to see, I had the same point of you before and then I realized I could search by folder name in VSCode and in Chromium so I've never add any real issue :)